keyboards/idb/idb_60/rules.mk

```diff
@@ -0,0 +1,30 @@
+# MCU name
+
+MCU = atmega32u2
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+
+BOOTLOADER = atmel-dfu
+
+# Build Options
+# Comment out to disable the options.
+
+BOOTMAGIC_ENABLE = lite
+MOUSEKEY_ENABLE = yes			# Mouse keys(+4700)
```

Comment:
```suggestion
MOUSEKEY_ENABLE = yes			# Mouse keys
```

Reply:
updated according to requested changes

Reply:
This one doesn't appear to be updated, yet.